plotting/plotter_3d.py

```
@@ -0,0 +1,375 @@
+        centroids = []
+
+        names = []
+        centroids = []
+        all_rings = []
+        data = None
+        for idx, a in enumerate(self.area):
+            if isinstance(a, str):
+
+                sp = a.split('/', 1)
+                if data is None:
+                    data = list_areas(sp[0], simplify=False)
```

Ruff (0.12.2)

Undefined name `list_areas`

(F821)